src/main/java/frc/team997/lib/trajectory/AsymmetricTrapezoidProfile.java

```diff
@@ -194,9 +140,6 @@ public AsymmetricTrapezoidProfile(Constraints constraints, State target, State i
                         ? Math.min(target.velocity, maxVelocity)
                         : Math.max(target.velocity, maxVelocity);
 
-        this.initial = new State(initial.position, initialVelocity);
```

Comment:
This can't be dropped - we use the initial velocity in the calculate function and we've recalculated the initialVelocity here